backend-base/src/main/java/org/eclipse/che/incubator/workspace/telemetry/base/AbstractAnalyticsManager.java

```diff
@@ -246,12 +247,12 @@ private Workspace getWorkspace(String endpoint) {
 
   }
 
-  private Long getDateFromString(String date) {
+  private Long getDateFromString(String fieldName, String date) {
     Long parsedDate = null;
     try {
       parsedDate = parseLong(date);
```

Comment:
should we have NPE check here e.g should we parse if date == 'null' ? also in which case date can be `null`?

Reply:
The date can be `null` if the workspace attribute is null, for example, if the workspace was just created.  From javadoc:

> NumberFormatException - if the string does not contain a parsable long.

so I think the only time it's throwing a `NumberFormatException` is when we have a case where the string is not a long or is null

Reply:
yeah, so my point is that maybe we should check if date is null and do not parse it, but rather log the message instead.